tests/test_writer.py

```diff
@@ -607,6 +607,32 @@ def test_add_named_destination(pdf_file_path):
         writer.write(output_stream)
 
 
+def test_add_named_destination_sort_order(pdf_file_path):
+    """
+    This test is regression test for issue #1927
```

Comment:
```suggestion
    Issue #1927 does not appear.
```

At some point I want to change our test runs that the first line of each test is shown when the test is running. Then this way of phrasing it is nicer :-)